components/maincontroller.js

```diff
@@ -199,7 +199,8 @@
         data.options = data.options || {};
         var cleanReceiverName = cleanName(data.receiverName || '');
         window.deviceInfo.deviceName = cleanReceiverName || window.deviceInfo.deviceName;
-        window.deviceInfo.deviceId = cleanReceiverName ? CryptoJS.SHA1(cleanReceiverName).toString() : window.deviceInfo.deviceId;
+        // deviceId just needs to be unique-ish
+        window.deviceInfo.deviceId = cleanReceiverName ? btoa(cleanReceiverName) : window.deviceInfo.deviceId;
```

Comment:
I wonder why we ever want to override it... looks like it's an internal property exposing serial number or something.

Reply:
For this particular we eventually will, by trial and error (that is, if it breaks because of this change we'll know that this change was wrong 🤣 ).

Reply:
Serial number or other IDs aren't accessible via js unfortunately